src/Algebra/FunctionProperties/Consequences.agda

```diff
@@ -126,6 +126,24 @@ assoc+id+invˡ⇒invʳ-unique {_•_} {_⁻¹} {ε} cong assoc id invˡ x y eq =
     (x ⁻¹) • ε       ≈⟨ proj₂ id (x ⁻¹) ⟩
     x ⁻¹             ∎
 
+------------------------------------------------------------------------
+-- Involution of inverses
+
+assoc+id+invʳ⇒invʳ-involutive : ∀ {_•_ _⁻¹ ε} →
+                                  Congruent₂ _•_ → Associative _•_ →
+                                  Identity ε _•_ → RightInverse ε _⁻¹ _•_ →
+                                  Involutive _⁻¹
+assoc+id+invʳ⇒invʳ-involutive {_•_} {_⁻¹} {ε} cong assoc id invʳ x =
+  sym (assoc+id+invʳ⇒invˡ-unique cong assoc id invʳ x (x ⁻¹) (invʳ _))
```

Comment:
I personally find this name `assoc+id+invʳ⇒invˡ-unique` from above is very bad, as the subject for unique is changed.